jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -576,6 +627,17 @@ private int evict()
             return getCapacity();
         }
 
+        private Statistics getStatistics()
+        {
+            List<Pool.Entry<RetainableByteBuffer>> entries = getPool().stream().toList();
+            int inUse = (int)entries.stream().filter(Pool.Entry::isInUse).count();
+            long pooled = _pooled.longValue();
+            long acquires = _acquires.longValue();
+            float hitRatio = acquires == 0 ? Float.NaN : pooled * 100F / acquires;
+            return new Statistics(getCapacity(), inUse, entries.size(), pooled, acquires, _releases.longValue(),
```

Comment:
Without the list, you can replace `entries.size()` with `getPool().size()` which is as cheap.